crates/next-custom-transforms/src/transforms/server_actions.rs

```diff
@@ -1064,26 +1064,6 @@ impl<C: Comments> VisitMut for ServerActions<C> {
         self.fn_decl_ident = old_fn_decl_ident;
     }
 
-    fn visit_mut_method_prop(&mut self, m: &mut MethodProp) {
-        let old_in_exported_expr = self.in_exported_expr;
-        let old_in_default_export_decl = self.in_default_export_decl;
-        self.in_exported_expr = false;
-        self.in_default_export_decl = false;
-        m.visit_mut_children_with(self);
-        self.in_exported_expr = old_in_exported_expr;
-        self.in_default_export_decl = old_in_default_export_decl;
-    }
-
-    fn visit_mut_class_method(&mut self, m: &mut ClassMethod) {
-        let old_in_exported_expr = self.in_exported_expr;
-        let old_in_default_export_decl = self.in_default_export_decl;
-        self.in_exported_expr = false;
-        self.in_default_export_decl = false;
-        m.visit_mut_children_with(self);
-        self.in_exported_expr = old_in_exported_expr;
-        self.in_default_export_decl = old_in_default_export_decl;
-    }
-
```

Comment:
were these just unused but not removed in a prior update?

Reply:
I just checked, they became obsolete with the PR below this one in the stack. I'll move the change over there...